packages/sample-plugin/README.md

```diff
@@ -0,0 +1,16 @@
+# Sample plugin
+
+### Docker config
+
+There's docker config with caddy server for easier build and run of the application. The Caddy config utilizes two environment variables
```

Comment:
@karelhala should we refer to this as the `plugin` and not an `application`? @vojtechszocs any input on the readme? I know you were going to look at adding them for the sample app and plugin.

Reply:
Good point! Will change it to `plugin`